crates/holochain_cascade/src/lib.rs

```diff
@@ -473,7 +470,6 @@ where
 
         // If we are not in the process of authoring this hash or its
         // authority we need a network call.
-        // TODO: do we want to put this behind an option, to allow cache-only queries?
```

Comment:
If we we want a cache-only query then just construct the cascade with only a cache database.